### PR TITLE
luci-proto-qmi: add PDP type option

### PR DESCRIPTION
**Pull request by nickberry17**
Signed-off-by: Nicholas Smith <nicholas.smith@telcoantennas.com.au>

### CODE REVIEW

File: protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js [hunk 1]
```diff
@@ -97,5 +97,11 @@ return network.registerProtocol('qmi', {
 		o = s.taboption('advanced', form.Value, 'mtu', _('Override MTU'));
 		o.placeholder = dev ? (dev.getMTU() || '1500') : '1500';
 		o.datatype    = 'max(9200)';
+		
+		o = s.taboption('general', form.ListValue, 'pdptype', _('PDP Type'));
+		o.value('ipv4v6', 'IPv4/IPv6 (both)');
```
Comment (feckert) [inline protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js]: Could you please translate this? The value **(both)** is not common for all language.

Comment (nickberry17) [reply protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js]: Good point, I guess I'll get line 71 as well?  I based it on that format.  Will commit tomorrow

Comment (feckert) [reply protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js]: I would say omit "both" because I think it is self-explanatory

Comment (nickberry17) [reply protocols/luci-proto-qmi/htdocs/luci-static/resources/protocol/qmi.js]: Done, but only for the IPv4/IPv6 value

**Comment by feckert** [comment]
Thanks merged

**Pull request merged**